### DISCUSSION TITLE
double数据精度问题

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#12588] [A - General / Q&A]
数据在落到doris之前产出值是1.14
字段设置double，落表后字段数据是1.1400000000000001
想问下这个要怎么处理

**Comment by JNSimba** [comment]
doris version？and how exactly does it work?

**Reply by hf200012** [reply]
这个会在1.2版本解决

**Reply by young138120** [reply]
> doris version？and how exactly does it work?

1.0.0-rc03

**Reply by young138120** [reply]
> 这个会在1.2版本解决

好的  感谢

**Comment by HappenLee** [comment]
https://github.com/apache/doris/pull/16190/files 

This pr will fix the problem
